_posts/2019-12-12-pulp-3-is-GA.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Pulp 3.0 is Generally Available
+author: Brian Bouterse
+tags:
+  - 3.0
+---
+The Pulp developer community is excited to announce the general availability of the following:
+
+* [pulp_rpm 3.0](https://pulp-rpm.readthedocs.io/en/3.0/) to manage RPM content
+* [pulp_container 1.0](https://pulp-container.readthedocs.io/en/1.0/) to manage Docker content
+* [pulp_file 0.1](https://pulp-file.readthedocs.io/en/0.1.0/) to manage Files
+
+These plugins are compatible with the [pulpcore 3.0.0](https://docs.pulpproject.org/) release which
+is also Generally Available today!
+
+### RPM Features
+
+pulp_rpm can sync, organize, and distribute the following content types currently:
+
+* RPMs
+* SRPMs
+* Advisory Content, i.e. Errata
+* Distribution Trees (kickstart trees)
+* Yum Metadata, e.g. comps.xml
+* modularity content
+
+You can upload the following to pulp_rpm:
+
+* RPM content
+* SRPM
+* Advisory
+
+### Container Features
+
+pulp_container can sync, organize, upload and distribute Docker content.
+
+### File Features
+
+pulp_file can sync, organize, upload, and distribute File content.
+
+
+## Installation
+
+See the installation docs (links below) for each plugin on how to install. They all use the same
+installer so they are very similar. Plugins are capable of being installed all together on one
+installation.
+
+* [pulp_rpm installation](https://pulp-rpm.readthedocs.io/en/3.0/installation.html)
+* [pulp_container installation](https://pulp-container.readthedocs.io/en/1.0/installation.html)
+* [pulp_file installation](https://pulp-file.readthedocs.io/en/0.1.0/installation.html)
+
+
+## Quickstart
+
+Pulp_rpm offers a [quickstart guide](https://pulp-rpm.readthedocs.io/en/3.0/quickstart.html) which
+uses [pulplift](https://github.com/pulp/pulplift) to create a not-long-living installation just to
+try it out. These same instructions can be adapted to the other plugins also.
+
+
+## Migrating from Pulp2 -> Pulp3
+
+A [Pulp 2 -> Pulp 3 migration tool is in-progress](https://github.com/pulp/pulp-2to3-migration),
+but not fully available yet. Until it is, users are recommended to:
+
+* **not** install Pulp3 on top of their Pulp2 system until it is available
+
+We hope this tool will be available within a month or two to migrate the plugin types that are
+released.
+
+
+## Why these version numbers?
+
+These Pulp plugins use [semantic versioning](https://semver.org/), so the different version numbers
+have different meanings summarized below.
+
+* pulp_rpm is 3.0 to indicate future releases will not contain API breaking changes, and it's the
+  next major release after pulp_rpm 2.y
+* pulp_container uses 1.0 to indicate future releases will not contain API breaking changes, but it
```

Comment:
s/it/it is/

Reply:
fixed in next PR https://github.com/pulp/pulpproject.org/pull/230